src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -33,6 +33,7 @@
     <% } %>
     <a href="/conversations">Conversations</a>
     <a href="/about.jsp">About</a>
+    <a href="/profile">Profile</a>
```

Comment:
I feel like the profile page should be the last page in the header and activity feed as the first. Because activity feed is a quick way to catch up with everything and then for more details they can go to the conversations page. I feel like logically profile should be placed rightmost..